<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>880</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>880</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>850</width>
     <height>850</height>
    </size>
   </property>
   <layout class="QGridLayout" name="main_layout">
    <item row="0" column="0">
     <widget class="QWidget" name="game" native="true">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>700</height>
       </size>
      </property>
      <layout class="QGridLayout" name="game_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Goudy Stout</family>
           <pointsize>60</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(172,255,244,1), stop:1 rgba(255,172,176,1));</string>
         </property>
         <property name="text">
          <string>SMART SUDOKU</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="options" native="true">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>90</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { background-color: rgb(203, 172, 255);
                      color: rgb(0, 0, 0);
                      border-radius: 10px; }
QPushButton::hover { background-color: rgb(255, 255, 255);
                      color: rgb(0, 0, 0); }
QPushButton::pressed { background-color: rgb(0, 0, 0);
                      color: rgb(255, 255, 255); }
QPushButton::disabled { background-color: rgba(255, 255, 255, 0.5);
                      color: rgb(0, 0, 0); }</string>
      </property>
      <layout class="QGridLayout" name="options_layout">
       <item row="0" column="1">
        <widget class="QPushButton" name="upload">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Upload a sudoku image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="real_time">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Real time sudoku solver</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="random_generator">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Generate a random game</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="instructions">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Instructions</string>
    </property>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="instructions"/>
   <addaction name="about"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
